scrapy/extensions/httpcache.py

```diff
@@ -220,6 +224,8 @@ def open_spider(self, spider):
         dbpath = os.path.join(self.cachedir, '%s.db' % spider.name)
         self.db = self.dbmodule.open(dbpath, 'c')
 
+        logger.debug("Using DBM cache storage in %(cachepath)s" % {'cachepath': dbpath}, extra={'spider': spider})
```

Comment:
Now you provide all attributes of the spider. I think only the name is needed (to prevent a very full log)